Altis_Life.Altis/core/functions/fn_numberText.sqf

```diff
@@ -1,28 +1,23 @@
 /*
     File: fn_numberText.sqf
-    Author: Karel Moricky
 
     Description:
-    Convert a number into string (avoiding scientific notation)
+    Convert a number into string (avoiding scientific notation) with comma formatting
 
     Parameter(s):
     _this: NUMBER
 
     Returns:
     STRING
 */
-private ["_number","_mod","_digots","_digitsCount","_modBase","_numberText"];
+params [["_number",0,[0]],"_return"]; 
```

Comment:
Unsure of the purpose of _return here?

Reply:
Ahh, forgot to delete that bit. Feel free to remove it.

It was originally just an alternative (lazy) way to define a variable as private.
`params["_return"];
_return = 0;`
is the same as
`private _return = 0;`